openbb_platform/core/openbb_core/app/model/obbject.py

```diff
@@ -39,6 +41,10 @@
 class OBBject(Tagged, Generic[T]):
     """OpenBB object."""
 
+    accessors: ClassVar[Set[str]] = set()
```

Comment:
Is there a better or more intuitive naming than accessors?

Reply:
I think the reason @montezdesousa picked that nomenclature was due to the fact `pandas` uses it.